com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -835,13 +835,15 @@ private void downloadSymbols(IProgress progress) throws IOException, CompileExce
         }
 
         for(String platform : platforms) {
-            String symbolsFilename = "";
+            String symbolsFilename = null;
             switch(platform) {
                 case "armv7-darwin":
                 case "arm64-darwin":
                 case "x86_64-darwin":
                     symbolsFilename = String.format("dmengine%s.dSYM.zip", variantSuffix);
                     break;
+                case "wasm-web":
+                    platform = "js-web";
```

Comment:
I surely looks weird. I think I'd like a code comment here :)

Reply:
I agree. I think I'll change and not download any symbols at all.